bokeh/core/has_props.py

```diff
@@ -529,28 +530,33 @@ def query_properties_with_values(self, query, include_defaults=True):
             dict : mapping of property names and values for matching properties
 
         '''
+        themed_keys = set()
         result = dict()
         if include_defaults:
             keys = self.properties()
         else:
-            keys = set(self._property_values.keys())
+            # TODO (bev) For now, include unstable default values. Things rely on Instances
+            # always getting serialized, even defaults, and adding unstable defaults here
+            # accomplishes that. Unmodified defaults for property value containers will be
+            # weeded out below.
+            keys = set(self._property_values.keys()) | set(self._unstable_default_values.keys())
```

Comment:
This was a discovery. I originally made it so no unmodified defaults were sent at all. But intentional or error, we currently rely on `Instances` always getting sent, even unmodified default ones, e.g. the default box overlay. Without this, there is no default box overlay. I'd like to change this behaviour but maybe its better to just keep it at this point.